src/Dotnet.Script.Core/ScriptRunner.cs

```diff
@@ -47,10 +47,9 @@ public async Task<TReturn> Execute<TReturn>(string dllPath, IEnumerable<string>
             submissionStates[0] = globals;
 
             var resultTask = method.Invoke(null, new[] { submissionStates }) as Task<TReturn>;
-            TReturn returnValue;
             try
             {
-                returnValue = await resultTask;
+                _ = await resultTask;
```

Comment:
Can't we just do `await resultTask` here?

Reply:
Yes, but a discard captures the _intention_ to deliberately ignore the return value _at the call site_. Just awaiting doesn't tell you if the ignorance was an omission or an intention.